src/main/resources/org/jenkinsci/plugins/docker/workflow/declarative/GlobalConfig/config.groovy

```diff
@@ -28,9 +28,7 @@ package org.jenkinsci.plugins.docker.workflow.declarative.GlobalConfig
 
 def f = namespace(lib.FormTagLib)
 
-f.section(title:_("Declarative Pipeline (Docker)")) {
```

Comment:
@car-roll Does the issue affect the global configuration as well? If not, I think this file could be left as-is. I would hope that sections work for the global config page or else things would get really messy.

Reply:
you're  right,  global config was  unaffected.  Reverted the  changes in  fd57e01b20df938f10351bc1d0e4373af7c2d860